### PR TITLE
Centers the Resume Preview When Resume is Found

### PR DESCRIPTION
**Pull request by kevinhehee**
image

### CODE REVIEW

File: components/PDFViewer.jsx [hunk 1]
```diff
@@ -15,6 +15,8 @@ const PDFViewer = ({ pdf }) => {
   };
 
   useEffect(() => {
+    pdf =
```
Comment (shahdivyank) [inline components/PDFViewer.jsx:18]: remove this

File: components/PDFViewer.jsx [hunk 2]
```diff
@@ -15,6 +15,8 @@ const PDFViewer = ({ pdf }) => {
...
+      "https://www.w3.org/WAI/ER/tests/xhtml/testfiles/resources/pdf/dummy.pdf";
     if (pdf) {
       if (pdf.endsWith(".pdf")) {
```
Comment (shahdivyank) [inline components/PDFViewer.jsx:21]: you can combine the if statements

Comment (shahdivyank) [reply components/PDFViewer.jsx:21]: This doesn't work for google drive links

File: pages/user/resume.js [hunk 3]
```diff
@@ -23,12 +23,13 @@ const ResumePage = () => {
...
         </p>
 
-        <div className="pt-3 flex flex-col flex-grow w-11/12 pb-1">
-          <div className="w-3/5 flex flex-col flex-grow">
-            <ResumeUpload resume={session?.user?.resume} />
-            <PDFViewer pdf={session?.user?.resume} />
-            <ResumeFeedback />
-          </div>
+        <div className="pt-3 flex flex-col w-11/12 pb-1">
+          <ResumeUpload resume={session?.user?.resume} />
```
Comment (shahdivyank) [inline pages/user/resume.js:27]: session?.user?.resume shouldnt be necessary. protected page handles it

File: components/PDFViewer.jsx (hunk 2 above)
Comment (kevinhehee) [reply components/PDFViewer.jsx:21]: fixed by searching for "drive.google.com" in string

File: components/PDFViewer.jsx (hunk 1 above)
Comment (kevinhehee) [reply components/PDFViewer.jsx:18]: accidental leftover, sorry about that

File: pages/user/resume.js (hunk 3 above)
Comment (kevinhehee) [reply pages/user/resume.js:27]: Same issue with 

https://github.com/acm-ucr/membership-portal/pull/489

image

File: components/PDFViewer.jsx (hunk 2 above)
Comment (kevinhehee) [reply components/PDFViewer.jsx:21]: image

File: components/PDFViewer.jsx [hunk 4]
```diff
@@ -15,8 +15,8 @@ const PDFViewer = ({ pdf }) => {
   };
 
   useEffect(() => {
-    if (pdf) {
-      if (pdf.endsWith(".pdf")) {
+    {
+      if (pdf && (pdf.endsWith(".pdf") || pdf.search("drive.google.com"))) {
```
Comment (shahdivyank) [inline components/PDFViewer.jsx:19]: you dont need any of these checks at the moment, you can just leave it at the assignment

File: pages/user/resume.js [hunk 5]
```diff
@@ -23,12 +23,13 @@ const ResumePage = () => {
...
         </p>
 
-        <div className="pt-3 flex flex-col flex-grow w-11/12 pb-1">
-          <div className="w-3/5 flex flex-col flex-grow">
-            <ResumeUpload resume={session?.user?.resume} />
-            <PDFViewer pdf={session?.user?.resume} />
-            <ResumeFeedback />
-          </div>
+        <div className="pt-3 flex flex-col w-11/12 pb-1">
+          <ResumeUpload resume={session?.user.resume} />
```
Comment (shahdivyank) [inline pages/user/resume.js:27]:
```suggestion
          <ResumeUpload resume={session.user.resume} />
```

File: pages/user/resume.js [hunk 6]
```diff
@@ -23,12 +23,13 @@ const ResumePage = () => {
...
+        </div>
+
+        <div className="w-3/5 flex flex-col flex-grow items-center">
+          <PDFViewer pdf={session?.user.resume} />
```
Comment (shahdivyank) [inline pages/user/resume.js:31]:
```suggestion
          <PDFViewer pdf={session.user.resume} />
```

**Review by shahdivyank — APPROVED** [review]

**Pull request merged**